### DISCUSSION TITLE
ND

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#6083] [General]
Hi developers...

experimental version:

EFIS range 80 

![grafik](https://user-images.githubusercontent.com/76531893/138765066-d2133b05-ccda-4ee6-b25a-6253bc4fd2bd.png)

EFIS range 160

![grafik](https://user-images.githubusercontent.com/76531893/138765126-4f938586-c9bf-4ca9-9b2e-3c21f4d82417.png)

just for info .... inbound KDFW with some strange routing in ND into RWY 11 ...

greets

**Comment by Benjozork** [comment]
I'm not sure what exactly the issue is.

**Comment by CptBayer** [comment]
Hi,

some waypoints are missing within the range of 160 NM respectively aren´t displayed ... WPT FDUBB for example ... 

greets ...
